OsmAnd/src/net/osmand/plus/views/MapLayers.java

```diff
@@ -257,12 +259,13 @@ public void updateLayers(@Nullable MapActivity mapActivity) {
 
 	public void updateMapSource(@NonNull OsmandMapTileView mapView, CommonPreference<String> settingsToWarnAboutMap) {
 		OsmandSettings settings = app.getSettings();
+		boolean useOpenGLRender = settings.USE_OPENGL_RENDER.get();
 
 		// update transparency
 		int mapTransparency = 255;
 		if (settings.MAP_UNDERLAY.get() != null) {
 			mapTransparency = settings.MAP_TRANSPARENCY.get();
-		} else if (settings.MAP_OVERLAY.get() != null) {
+		} else if (useOpenGLRender && settings.MAP_OVERLAY.get() != null) {
```

Comment:
I don't think this code was changed since it was working in Legacy, looks strange

Reply:
This code wasn't present in Legacy